server/sources.go

```diff
@@ -451,7 +451,7 @@ func (s *Service) SourceUserID(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 	store := ts.Users(ctx)
-	u, err := store.Get(ctx, uid)
+	u, err := store.Get(ctx, chronograf.UserQuery{Name: &uid})
```

Comment:
1. see above comment about `Name` being an insufficient query due to non-uniqueness applies in `server/sources`, if the Bolt store is shared. disregard if irrelevant.
1. also, it feels unsemantic to search for `Name` with a `uid` – is this possible to make consistent?

Reply:
> see above comment about Name being an insufficient query due to non-uniqueness applies in server/sources, if the Bolt store is shared. disregard if irrelevant.

for InfluxDB/Plutonium users name is guaranteed to be unique. It's only for the boltdb users that they're actually different.

> also, it feels unsemantic to search for Name with a uid – is this possible to make consistent?

the `:uid` is coming from the mux

```
router.GET("/chronograf/v1/sources/:id/users/:uid", EnsureViewer(service.SourceUserID))
```

we could change it to name if you'd like.


I think the issue is that in InfluxDB/Pu land `User.Name` is a unique identifier, but in our BoltDB user land `User.ID` is a unique identified. Each implementation of the `UsersStore` has slightly different semantics and I think thats expected.